wsclient/src/main/java/org/hpccsystems/ws/client/HPCCWsWorkUnitsClient.java

```diff
@@ -888,6 +888,17 @@ public WorkunitInfo compileWUFromECL(WorkunitInfo wu ) throws Exception
             createdWU.setCluster(wu.getCluster());
             submitWU(createdWU); // if no exception proceed
             this.monitorWUToCompletion(createdWU);
+
+            //exceptions, etc. aren't always included in the submit response; do another request to get all workunit info
+            WUInfo parameters=new WUInfo();
```

Comment:
I wish we could build this into monitorWUToCompletion since that method is already calling WUInfo